src/scripts/interns.js

```diff
@@ -19,7 +19,13 @@ async function pairInterns() {
     }
   }
   interns = filterByDepartment(filterByLocation(interns));
+
+  const selected = getSelectedOptions()["Unique Pairing"];
+
   shuffle(interns);
+  if (selected.length !== 0) {
+    return uniquePairing(interns, selected);
+  }
```

Comment:
Actually, we can make `uniquePairing` a void and always handle the paring in this block here
```suggestion
  shuffle(interns);
  uniquePairing(interns);
```